pkg/plugin/conftest/plugin_test.go

```diff
@@ -819,6 +818,35 @@ func TestPlugin_ConfigHash(t *testing.T) {
 func TestPlugin_GetContainerName(t *testing.T) {
 	g := NewGomegaWithT(t)
 
-	plugin := conftest.NewPlugin(ext.NewSimpleIDGenerator(), fixedClock)
+	plugin := NewPlugin(ext.NewSimpleIDGenerator(), fixedClock)
 	g.Expect(plugin.GetContainerName()).To(Equal("conftest"))
 }
+
+func TestGetPolicyTitleFromResult(t *testing.T) {
+	a := &plugin{idGenerator: ext.NewGoogleUUIDGenerator()}
+	tests := []struct {
+		name   string
+		result Result
+		equal  bool
+		want   string
+	}{
+		{name: "get Policy ID From Result", result: Result{Message: "aaa", Metadata: map[string]interface{}{"id": "1234"}}, want: "1234", equal: true},
+		{name: "get Policy Title From Result", result: Result{Message: "bbb", Metadata: map[string]interface{}{"title": "title 1234"}}, want: "title 1234", equal: true},
+		{name: "get Policy ig generator From Result", result: Result{Message: "ccc", Metadata: map[string]interface{}{}}, want: "", equal: false}}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got := a.getPolicyTitleFromResult(tt.result)
```

Comment:
Testing private methods is usually where bigger problems with testability and well defined APIs start. We should validate the logic implemented in the getPolicyTitleFromResult method by exercising public APIs.